modules/local/yahs.nf

```diff
@@ -0,0 +1,44 @@
+process YAHS {
```

Comment:
Can we please push this to nf-core? This is our software, so it would great if others picked it up as well. And nf-core/genomeassember would be a good candidate for that.